uninstall.php

```diff
@@ -9,5 +9,7 @@
 
 require_once dirname( __FILE__ ) . '/woocommerce-admin.php';
 
-WC_Admin_Feature_Plugin::instance()->includes();
-WC_Admin_Install::delete_table_data();
+if ( WC_Admin_Feature_Plugin::instance()->check_dependencies() ) {
```

Comment:
It looks like `WC_Admin_Install::delete_table_data()` has no dependencies - perhaps we can just include its source file and call it?

Reply:
It does have dependencies:

```
PHP Fatal error:  Uncaught Error: Class 'WC_Admin_Install' not found in /wp-content/plugins/woocommerce-admin/uninstall.php:15
```

Reply:
> It does have dependencies:

Ignore that. I was relying on the autoloader. Addressed in https://github.com/woocommerce/woocommerce-admin/pull/2762/commits/8676faee9e2aeecdd1c054fff31d67af33a08c17